Symfony/CS/Tests/Fixer/PSR2/UnneededControlParenthesesFixerTest.php

```diff
@@ -0,0 +1,329 @@
+            array(
+                '<?php
+                echo(1 + 2) * 10, "\n";
+                ',
+                '<?php
+                echo((1 + 2) * 10, "\n");
+                ',
+            ),
+            array(
+                '<?php echo "foo" ?>',
+                '<?php echo ("foo") ?>',
+            ),
+            array(
+                '<?php print "foo" ?>',
+                '<?php print ("foo") ?>',
+            ),
+            array(
+                '<?php
+                echo "foo";
+                print "foo";
+                ',
+                '<?php
+                echo ("foo");
+                print ("foo");
+                ',
+            ),
+            array(
+                '<?php
+                echo"foo";
+                print"foo";
+                ',
+                '<?php
+                echo("foo");
+                print("foo");
+                ',
+            ),
+            array(
+                '<?php
+                echo $a ? $b : $c;
+                echo ($a ? $b : $c) ? $d : $e;
+                echo 10 * (2 + 3);
+                echo ("foo"), ("bar");
+                echo my_awesome_function("foo");
+                echo $this->getOutput(1);
+                ',
+                '<?php
+                echo ($a ? $b : $c);
+                echo ($a ? $b : $c) ? $d : $e;
+                echo 10 * (2 + 3);
+                echo ("foo"), ("bar");
+                echo my_awesome_function("foo");
+                echo $this->getOutput(1);
+                ',
+            ),
+            array(
+                '<?php
+                return "prod";
+                ',
+            ),
+            array(
+                '<?php
+                return (1 + 2) * 10;
+                ',
+            ),
+            array(
+                '<?php
+                return (1 + 2) * 10;
+                ',
+                '<?php
+                return ((1 + 2) * 10);
+                ',
+            ),
+            array(
+                '<?php
+                return "prod";
+                ',
+                '<?php
+                return ("prod");
+                ',
+            ),
+            array(
+                '<?php
+                return$x;
+                ',
+                '<?php
+                return($x);
+                ',
+            ),
+            array(
+                '<?php
+                switch ($a) {
+                    case "prod":
+                        break;
+                }
+                ',
+            ),
+            array(
+                '<?php
+                switch ($a) {
+                    case "prod":
+                        break;
+                }
+                ',
+                '<?php
+                switch ($a) {
+                    case ("prod"):
+                        break;
+                }
+                ',
+            ),
+            array(
+                '<?php
+                case$x;
```

Comment:
IMO it should be `case $x` (same fore other keywords).
And please add test case for `case(2)`. `case2` would be invalid.